sdk/tools/azure-openrewrite/src/test/java/com/azure/openrewrite/migration/FullSampleMigrationTest.java

```diff
@@ -65,20 +78,54 @@ static Stream<Path> gatherAllRegularFiles(Path sampleDir) throws IOException {
             .filter(Files::isRegularFile).collect(Collectors.toList()).stream();
     }
 
+
+
+    private static List<NamedStyles> getStyles() {
```

Comment:
What are these styles?

Reply:
These were needed to format the code for consistency in the test runs.